the-collab-lab/tcl-17-smart-shopping-list
# Add textField, spacing between rest button

## Description
This PR adds styles using the Material UI `textField` component for the search box.



## Related Issue

closes #56


## Acceptance Criteria

- [x] Create space between label, input element, and button
- [x] Fix accessibility for `reset` button and `delete` icon


## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|  ✓ |:hammer: Refactoring       |

## Updates

### Before
<img width="302" alt="Screen Shot 2021-02-26 at 1 18 13 PM" src="https://user-images.githubusercontent.com/14856004/109345013-225c3480-7835-11eb-91ea-da303b7d11ea.png">





### After
<img width="455" alt="Screen Shot 2021-02-26 at 1 10 47 PM" src="https://user-images.githubusercontent.com/14856004/109344401-48350980-7834-11eb-8693-f91bc293c846.png">





## Testing Steps / QA Criteria
* One way to test the site is to click on the `details` button below to open the `Netlify preview`

![Screen Shot 2021-01-12 at 6 05 51 PM](https://user-images.githubusercontent.com/14856004/104389400-ff81f500-5500-11eb-95ae-4c0169a069cb.png)

* Otherwise from your terminal, pull down this branch with `git pull origin VT-list-search-filter-spacing` and check that branch out with `git checkout VT-list-search-filter-spacing`
* Create a new list or join an existing list
* The list shown uses the token `stroll ames knack`
